src/gc/impl/conservative/gc.d

```diff
@@ -1131,12 +1131,24 @@ class ConservativeGC : GC
     }
 
 
+    void collect(bool block = true) nothrow
+    {
+        fullCollect(block);
+    }
+
+
     void collect() nothrow
```

Comment:
Why not just extending this one? Aren't both conflicting when `collect()` is called since the new function has a default value?